media-libs/libavif/libavif-0.8.0.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="Library for encoding and decoding .avif files"
+DESCRIPTION="This library aims to be a friendly, portable C implementation of \
+the AV1 Image File Format, as described here: \
+ \
+https://aomediacodec.github.io/av1-avif/ \
+ \
+It is a work-in-progress, but can already encode and decode all AOM supported \
+YUV formats and bit depths (with alpha). \
+ \
+For now, it is recommended that you checkout/use tagged releases instead of \
+just using the master branch. I will regularly create new versions as \
+bugfixes and features are added."
```

Comment:
IMHO this looks better when opening with HaikuDepot:
```
DESCRIPTION="This library aims to be a friendly, portable C implementation of \
the AV1 Image File Format, as described here: \
https://aomediacodec.github.io/av1-avif/ 

It is a work-in-progress, but can already encode and decode all AOM supported \
YUV formats and bit depths (with alpha).

For now, it is recommended that you checkout/use tagged releases instead of \
just using the master branch. I will regularly create new versions as \
bugfixes and features are added."
```

Reply:
Thanks!